File: Number Series/Basic/Division.java
import java.io.*;
import java.lang.System.*;

class Program {
	public static void main(String args[]) {

		BufferedReader br = new BuferedReader (new InputStreamReader(System.in));

		int num1 = Integer.parseInt(br.readLine());
		int num2 = Integer.parseInt(br.readdLine());
		
		out.println("Multiplication : "+ (num1 * num2));

		if(num1 > num2)	
			out.println("Division: "+ (num1 / num2));
		else
			out.println("Division : "+ (num2 / num1));
		

	}
}

File: Number Series/Basic/Division.c
#include<stdio.h>

void main() {

	int num1,num2;
	printf("Enter number 1: ");
	scanf("%d",&num1);

	printf("Enter number 2: ");
	scanf("%d",&num2);
	
	//multiplication
	print("Multiplication = %d\n",num1*num2);

	//division
	if(num1 > num2)
		print("Division = %d\n",num1/ num2);
	else
		print("Division = %d\n",num2/num1);
}


